hedera-node/src/main/java/com/hedera/services/store/models/Account.java

```diff
@@ -125,6 +127,10 @@ public CopyOnWriteIds getAssociatedTokens() {
 		return associatedTokens;
 	}
 
+	public Address getAddress() {
```

Comment:
nit: I think that it would be better if we rename this method to `getEvmAddress`